pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[build-system]
+requires = ["setuptools", "wheel", "versioneer"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "MembraneCurvature"
+version = "1.0.0"
+license = {file = "LICENSE"}
+description = "MDAnalysis tool to calculate membrane curvature from MD simulations."
+authors = [
+    {name = "Estefania Barreto-Ojeda", email = "estefania@ojeda-e.com"},
+]
+maintainers = [
+    {name = 'MDAnalysis Core Developers', email = 'mdanalysis@numfocus.org'}
+]
+readme = "README.md"
+requires-python = ">=3.9"
+keywords = [
+    "python", "biophysics", "molecular-dynamics",
+    "molecular-simulation", "analysis",
+    "trajectory-analysis",
+]
+dependencies = [
+    'numpy>=1.20.0',
+    'mdanalysis>=2.0.0'
+    ]
+
+classifiers = [
+        'Development Status :: 5 - Production/Stable',
+        'Environment :: Console',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows',
+        'Operating System :: POSIX',
+        'Topic :: Scientific/Engineering ',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12'
+]
+
+[project.optional-dependencies]
+tests = [
+    "pytest",
+    "pytest-cov",
+    "ipython",
+    "MDAnalysisTests"
+]
+docs = [
+    "ipython",
+    "MDAnalysisTests",
+    "sphinx_rtd_theme",
```

Comment:
todo: there's an MDAnalysis sphinx theme that we've been moving folks over to (it's more up to date) - maybe we can do that as a separate PR?

Reply:
yes, good idea. See #120